Kconfig

```diff
@@ -1956,6 +1956,19 @@ config DEBUG_OPTLEVEL
 		This string represents the custom optimization level that will be
 		used if DEBUG_CUSTOMOPT.
 
+config DEBUG_OPT_UNUSED_SECTIONS
+	bool "Optimization to eliminate the unused input sections"
+	default y
+	depends on ARCH_TOOLCHAIN_GNU
```

Comment:
remove ARCH_TOOLCHAIN_GNU, since ez80 support the data/function section but it isn't gcc derived.